index.html

```diff
@@ -3257,15 +3257,14 @@ <h2>Syntaxes</h2>
 <a href="#advanced-concepts"></a> is the canonical structural representation of
 a <a>verifiable credential</a> or <a>verifiable presentation</a>. All
 serializations are representations of that data model in a specific format. This
-section specifies how the data model is realized in JSON-LD and plain JSON.
-Although syntactic mappings are provided for only these two syntaxes,
+section specifies how the data model is realized in JSON-LD for the base
```

Comment:
what does the word "base" do for us here? feels a bit confusing.

Reply:
I think it reads the same without the word 'base'
I'd support removing.

Reply:
'base' has been removed in 654b694fd41580ba7724d1593f5c1906de3180e5